libs/knowledge-store/notebooks/astra_support.ipynb

```diff
@@ -125,10 +105,11 @@
     "from typing import AsyncIterator, Iterable\n",
     "from ragstack_knowledge_store.graph_store import CONTENT_ID\n",
     "from markdownify import MarkdownConverter\n",
-    "from ragstack_langchain.graph_store.extractors import HtmlLinkEdgeExtractor\n",
+    "from ragstack_langchain.graph_store.links import add_links\n",
```

Comment:
I wonder if we should tinker with the `__init__.py` so this can be `from ragstack_langchain.graph_store import add_links` or even make it part of `extractors`?

Reply:
@cbornet thoughts on naming or just moving `add_links` into extractors?

Reply:
Yes, I think we can put it in `__init__`.